modules/lang-mustache/src/internalClusterTest/java/org/elasticsearch/script/mustache/MultiSearchTemplateIT.java

```diff
@@ -175,10 +176,9 @@ public void testBasic() throws Exception {
         assertThat(response4.getFailure().getMessage(), equalTo("no such index [unknown]"));
 
         MultiSearchTemplateResponse.Item response5 = response.getResponses()[4];
-        assertThat(response5.isFailure(), is(false));
-        SearchTemplateResponse searchTemplateResponse5 = response5.getResponse();
-        assertThat(searchTemplateResponse5.hasResponse(), is(false));
-        assertThat(searchTemplateResponse5.getSource().utf8ToString(), equalTo("{\"query\":{\"terms\":{\"group\":[1,2,3,]}}}"));
+        assertThat(response5.isFailure(), is(true));
```

Comment:
This looks like a BWC break, but in fact this request would always throw an error via a REST request because the RestToXContentListener would try and parse things, but this test goes via the internal client which doesn't do any json parsing.

Reply:
ack